content/projects/gsoc/students.adoc

```diff
@@ -50,31 +81,203 @@ We highly recommend to make some contributions to the project while you work on
 It will help you to polish the proposal,
 and mentors will consider contributions and interactions with the community when processing applications.
 
-Useful links:
+[[UsefulLinks]]
+=== Useful links
+
+Here are a list of links to help you get started on participating in Jenkins and in coding for the Jenkins project,
+in increasing level of difficulty.
+
+* link:https://jenkins.io/project/governance/[Project Governance]
```

Comment:
Links should be relative, e.g. `/project/governance/`

Reply:
Fixed this one and all similar ones, will push momentarily.